scripts/update_active_context.py

```
@@ -0,0 +1,158 @@
+import os
```

Ruff (0.15.2)

[error] The file is executable but no shebang is present

(EXE002)